journalist_gui/journalist_gui/SecureDropUpdater.py

```diff
@@ -13,6 +15,25 @@
 ESCAPE_POD = re.compile(r'\x1B\[[0-?]*[ -/]*[@-~]')
 
 
+def prevent_second_instance(app: QtWidgets.QApplication, name: str) -> None:  # noqa
```

Comment:
Unlike the client, we don't use type hints in the updater GUI, even though it is just supported by the version of Python here. But I see no harm in having these here